sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -409,3 +409,41 @@ Editor type `select` allows the user to pick items from a select, providing mult
 ```
 
 To correctly define options for multiselect, you need to define the `items` property and then provide values and (optionally) labels in `enum` and `enumTitles` properties.
+
+### Resource
+
+The Resource property enables Actor users to select different types of resources.
```

Comment:
I'd say that:

Resource type identifies what kind of object is referred to in the input field. For example, a the Key-value store resource type can be referred to using a string ID.